projects/plugins/jetpack/_inc/client/admin.js

```diff
@@ -125,6 +128,8 @@ export function getRouteName( path ) {
 			return _x( 'Settings', 'Navigation item.', 'jetpack' );
 		case '/discussion':
 			return _x( 'Discussion', 'Navigation item.', 'jetpack' );
+		case '/earn':
```

Comment:
FYI we will rename the section in WP.com soon to "Monetize" in upcoming projects.

We can rename it simultaneously across Jetpack and WP.com then!